alfresco-transform-misc/alfresco-transform-misc-spring/src/test/java/org/alfresco/transformer/MiscControllerTest.java

```diff
@@ -62,7 +62,6 @@
 
 @RunWith(SpringRunner.class)
 @WebMvcTest(MiscController.class)
-@Import({SelectingTransformer.class})

```

Comment:
I assume this was an intentional removal ?

Reply:
@eknizat

Reply:
Yes, intentional